sound/hybrid_font.h

```diff
@@ -186,7 +191,13 @@ class HybridFont : public SaberBase {
     // Radians per second per second
     float rss = sqrtf(gyro_slope.z * gyro_slope.z + gyro_slope.y * gyro_slope.y) * (M_PI / 180);
     float swing_speed = fusor.swing_speed();
-    if (swing_speed > swingThreshold) {
+    uint32_t now = micros();
+    uint32_t delta_micros = now - last_swing_micros_;
+    last_swing_micros_ = now;
+    float delta = delta_micros * 0.000001;
+    angle_ += 0.98 * gyro.len() * delta;
+    angle_ += 0.02 * fusor.accel().len();
+      if (swing_speed > swingThreshold) {
```

Comment:
Stupid github won't let me put the comment in the right place....
Anyways, I think we need to zero out angle_ when we're not swinging, which would be at the other branch of this if statement.

Reply:
done